web/src/search_pill.ts

```diff
@@ -0,0 +1,49 @@
+import {Filter} from "./filter";
+import * as input_pill from "./input_pill";
+import type {InputPillContainer} from "./input_pill";
+
+type SearchPill = {display_value: string; type: string; description_html: string};
```

Comment:
I would wrap this with each value on its own line for better readability.

Reply:
Oh weird yeah I expected the linter to do that. Fixed